crates/uv-resolver/src/requires_python.rs

```diff
@@ -472,6 +455,36 @@ impl RequiresPython {
             }
         })
     }
+
+    /// Return a set of fork preferences to cover the range of supported Python minor versions.
+    pub fn forks(&self) -> Vec<MarkerTree> {
+        let mut forks = vec![];
+
+        let python_marker = self.to_marker_tree();
+
+        // Add, e.g., `python_full_version == '3.8.*'` for all supported Python versions.
+        for minor in 7..13 {
+            let python_version = Version::new([3, minor]);
+            let marker = MarkerTree::expression(MarkerExpression::Version {
+                key: MarkerValueVersion::PythonFullVersion,
+                specifier: VersionSpecifier::equals_star_version(python_version),
+            });
+            if !marker.is_disjoint(&python_marker) {
+                forks.push(marker);
+            }
+        }
+
+        // Add, e.g., `python_full_version >= '3.13'` for the latest supported Python version.
+        let marker = MarkerTree::expression(MarkerExpression::Version {
+            key: MarkerValueVersion::PythonFullVersion,
+            specifier: VersionSpecifier::greater_than_equal_version(Version::new([3, 13])),
+        });
+        if !marker.is_disjoint(&python_marker) {
+            forks.push(marker);
+        }
+
+        forks
```

Comment:
@konstin -- Re-requesting review as I changed this setting to _also_ automatically solve for each Python minor.